public/app/features/manage-dashboards/utils/validation.ts

```diff
@@ -4,8 +4,29 @@ import { validationSrv } from '../services/ValidationSrv';
 
 export const validateDashboardJson = (json: string) => {
   try {
-    JSON.parse(json);
-    return true;
+    let dashboard = JSON.parse(json);
+    // If dashboard.tags provided && is Array
```

Comment:
The comments don't add any new information and the code is easy to read, so what do you think about having no comments? It seems to just create twice as many lines